### DISCUSSION TITLE
Node-Red OpenID Connect help

### DISCUSSION THREAD

**Discussion by schklom** [authelia/authelia#5509] [Q&A] [answered]
Authentik has a tutorial to login with OIDC to Node-Red ([here](https://goauthentik.io/integrations/services/node-red/#step-4)), but I am not sure how to adapt the URLs to Authelia.
The part I am unsure about is
```js
        options: {
                issuer: 'https://authentik.company/application/o/<application-slug>/',
                authorizationURL: 'https://authentik.company/application/o/authorize/',
                tokenURL: 'https://authentik.company/application/o/token/',
                userInfoURL: 'https://authentik.company/application/o/userinfo/',
```
If my Authelia instance is at https://authelia.company, can I simply substitute authentik.company with authelia.company?

Ideally, if someone finds a way to make it work, Node-Red could be added to the list https://www.authelia.com/integration/openid-connect/

**Comment by james-d-elliott** [comment, marked as answer]
Our endpoints are listed here: https://www.authelia.com/integration/openid-connect/introduction/#discoverable-endpoints, the paths will be a suffix of https://auth.example.com if that's your domain.

**Reply by schklom** [reply]
Thank you so much, it works now :)

**Comment by schklom** [comment]
To anyone else who had issues with this, the `issuer` is the domain (e.g. https://auth.example.com).

**Comment by pepex7** [comment]
@schklom Could you guide me in integrating Authelia with Node-Red? I have been trying for a long time without success.

https://discourse.nodered.org/t/dashboard-authelia-auth/89784/28

**Reply by james-d-elliott** [reply]
I've drafted this rough integration guide for OIDC: https://deploy-preview-7657--authelia-staging.netlify.app/integration/openid-connect/node-red/
